drivers/pwm/pwm_nrf5_sw.c

```diff
@@ -7,11 +7,21 @@
 #include <soc.h>
 
 #include "pwm.h"
+#include <nrf_peripherals.h>
 
 #define LOG_LEVEL CONFIG_PWM_LOG_LEVEL
 #include <logging/log.h>
 LOG_MODULE_REGISTER(pwm_nrf5_sw);
 
+/* One compare channel is needed to set the PWM period, hence +1. */
```

Comment:
This comment refers to line [18](https://github.com/zephyrproject-rtos/zephyr/pull/12390/files#diff-0f7fc8bd683a80e71bb5665dfca79dfbR18) (`+ 1` occurs there).
I think the best would be to move the line with `#define` after the check (first validate, then use).

Reply:
Agreed. moved.